frontend/src/components/activity/ScheduleEntry.vue

```diff
@@ -93,7 +93,7 @@ Displays a single scheduleEntry
         <template v-else>{{ $tc('global.button.back') }}</template>
       </v-btn>
 
-      <TogglePaperSize :value="isPaperDisplaySize" @input="toggleDisplaySize" />
```

Comment:
Why did you remove this functionality? This prevents a layout jump when you toggle the displaySize, because it changes the responsiveLayout to be papersize before the animation takes place.

Reply:
I had to remove the previous local storage implementation, which was done in the composable useDisplaySize.js. When doing this, I overlooked the `nextTick` in there. But I am mostly convinced it does not make any perceivable difference. If it does, can you tell me which one of these screencasts comes from dev (with `nextTick`) and which one comes from my local instance (without `nextTick`)?

[Blind Test A.webm](https://github.com/ecamp/ecamp3/assets/7566995/aacd3808-23e4-44fd-af42-448a036ac780)
[Blind Test B.webm](https://github.com/ecamp/ecamp3/assets/7566995/b8768271-2cbf-4048-aa8e-0a6967b6a5d6)

However, previously the paper size display of the responsive layout was broken in this PR. I fixed this now in [82f2a9c](https://github.com/ecamp/ecamp3/pull/4559/commits/82f2a9cd9fc912ac671103f643f83c8c5e2c69a2).

Reply:
Hmm, shouldn't this be a computed and not a function in [82f2a9c](https://github.com/ecamp/ecamp3/pull/4559/commits/82f2a9cd9fc912ac671103f643f83c8c5e2c69a2)?
Maybe this was required with the initial local value variant.

Reply:
I removed the computed when I switched from the composition API paradigms back to the options API way for this feature.
What advantages do you expect from using a composition API computed here over a simple function like we do for the [other provided values](https://github.com/ecamp/ecamp3/blob/82f2a9cd9fc912ac671103f643f83c8c5e2c69a2/frontend/src/components/activity/ScheduleEntry.vue#L276..L278)? I did try passing down the plain value, but that [isn't reactive](https://v2.vuejs.org/v2/api/#provide-inject). The Vue 3 docs explicitly [recommend](https://vuejs.org/guide/components/provide-inject#working-with-reactivity) using a computed for this case, but it seemed inconsistent with our other provides, and I didn't want to touch all other provides. But if you insist on using computed for just this one provide, I can change it.

Reply:
I don't mind the implementation. I just hope that it won't break at random :D